bob-access/src/authenticator/users_storage/config.rs

```diff
@@ -26,7 +37,8 @@ pub(super) fn parse_users(
         let &perms = roles
             .get(&u.role)
             .ok_or_else(|| Error::Validation(format!("Can't find role {}", u.role)))?;
-        let user = User::new(u.username.clone(), u.password, perms);
+        let hash = u.password_hash.map(|h| Vec::from_hex(h).expect("Invalid sha512 hash"));
```

Comment:
Add additional check, that `password_hash` has 64 bytes size (512 / 8)

Reply:
Code already will panic if hash's length is not 512 bits